collections/luci-ssl/Makefile

```diff
@@ -9,8 +9,8 @@ include $(TOPDIR)/rules.mk
 LUCI_TYPE:=col
 LUCI_BASENAME:=ssl
 
-LUCI_TITLE:=LuCI with HTTPS support (mbedTLS as SSL backend)
-LUCI_DEPENDS:=+luci +libustream-mbedtls +px5g
+LUCI_TITLE:=LuCI with HTTPS support (WolfSSL as SSL backend)
+LUCI_DEPENDS:=+luci +libustream-mbedtls +px5g-wolfssl
```

Comment:
You are still requiring both mbedtls and wolfssl to be included, as you did NOT change the SSL backend (libustream) itself.

``` 
+libustream-wolfssl +px5g-wolfssl
```

Reply:
You could also remove the px5g part from the commit title, as this should handle both the SSL backend and the px5g side at the same time.

Reply:
Done, thanks.